x-pack/plugins/ingest_pipelines/server/routes/api/get.ts

```diff
@@ -18,33 +18,22 @@ const paramsSchema = schema.object({
 export const registerGetRoutes = ({
   router,
   license,
-  lib: { isEsError },
+  lib: { handleEsError },
 }: RouteDependencies): void => {
   // Get all pipelines
   router.get(
     { path: API_BASE_PATH, validate: false },
     license.guardApiRoute(async (ctx, req, res) => {
-      const { callAsCurrentUser } = ctx.core.elasticsearch.legacy.client;
+      const { client: clusterClient } = ctx.core.elasticsearch;
 
       try {
-        const pipelines = await callAsCurrentUser('ingest.getPipeline');
+        const { body: pipelines } = await clusterClient.asCurrentUser.ingest.getPipeline({
```

Comment:
I don't think an `id` is required or needed to fetch all pipelines.

I'm actually getting an error currently when trying to load the list view as-is. (I'm surprised a test didn't catch this 🤔 )

<img width="1104" alt="Screen Shot 2021-04-07 at 1 35 25 PM" src="https://user-images.githubusercontent.com/5226211/113910941-6254f680-97a7-11eb-8398-2ba8a9de17a1.png">

Reply:
Haha this is what happens when you commit test code! This is how I came across the issue with messages returned in `handlEsError` 😅 . Will fix!